oauth-client/src/main/java/io/strimzi/kafka/oauth/client/JaasClientOauthLoginCallbackHandler.java

```diff
@@ -81,7 +83,7 @@ public void configure(Map<String, ?> configs, String saslMechanism, List<AppConf
             clientSecret = config.getValue(Config.OAUTH_CLIENT_SECRET);
 
             if (clientId == null) {
-                throw new RuntimeException("No client id specified ('oauth.client.id')");
+                throw new RuntimeException("No client id specified (OAUTH_CLIENT_ID)");
```

Comment:
Same as above.

Reply:
All documentation everywhere only really mentions env vars, and in some other places there already was env var used in error messages. I opted to use env vars everywhere in error messages for consistency, rather than using equivalent system property names.